styles/TextResponse.module.css

```diff
@@ -23,3 +23,14 @@
   border: 3px solid #333;
   outline: none;
 }
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  gap: 20px;
+  font-size: 20px;
+  border: none;
+  overflow: scroll;
+  -ms-overflow-style: none; /* Internet Explorer 10+ */
+  scrollbar-width: none; /* Firefox */
```

Comment:
It looks like a lot of these were copied over from the radio button container, and aren't relevant to the text box (eg. a lot of the flex elements). Maybe remove those that aren't applicable? But I will say I will overwrite a lot of these in #19!

Reply:
Can confirm these were all copied from the radio button container. I can definitely remove the unnecessary attributes before merging

Reply:
Okay turns out only three attributes are needed I think. Thanks for catching that; I almost pushed font size 20px 😨